### PR TITLE
Upgrade Log4j for Security

### PR DESCRIPTION
**Pull request by RickHawesUSDS**
This PR upgrades Log4j because of a security vulnerability found by DependaBot. 

## Changes
- Log4j upgrade to 2.13+ 

## Checklist
- [X] Tested locally?
- [ ] Added new dependencies?
- [ ] Updated the docs?
- [ ] Added a test?
- [ ] Did you check for sensitive data, and remove any?
- [ ] Are additional approvals needed for this change?
- [ ] Are there potential vulnerabilities or licensing issues with any new dependencies introduced?

### CODE REVIEW

File: prime-router/pom.xml [hunk 1]
```diff
@@ -30,7 +30,7 @@
         <jackson.version>2.11.3</jackson.version>
         <hapi.version>2.3</hapi.version>
         <jooq.version>3.14.3</jooq.version>
-        <log4j.version>2.8.2</log4j.version>
+        <log4j.version>[2.13.2,)</log4j.version>
         <timestamp>${maven.build.timestamp}</timestamp>
```
Comment (jimduff-usds) [inline prime-router/pom.xml:34]: @RickHawesUSDS , this looks like a typo?

Comment (MauriceReeves-usds) [reply prime-router/pom.xml:34]: Yeah, I can't find any good documentation on the rules for syntax for version numbers in Maven, but this doesn't look correct.

Comment (RickHawesUSDS) [reply prime-router/pom.xml:34]: Here's some docs https://www.baeldung.com/maven-dependency-latest-version. 

This is not a typo. What this means is >= 2.13.2. It came from the DependaBot folks.

Comment (jimduff-usds) [reply prime-router/pom.xml:34]: It still _looks like_ a typo!   ;)

**Review by jimduff-usds — APPROVED** [review]

**Review by MauriceReeves-usds — APPROVED** [review]

**Review by ronaldheft-usds — APPROVED** [review]

**Pull request merged**